src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/ContentLengthReadStream.cs

```diff
@@ -212,11 +212,18 @@ public override async ValueTask<bool> DrainAsync(int maxDrainBytes)
                 CancellationTokenSource? cts = null;
                 CancellationTokenRegistration ctr = default;
                 TimeSpan drainTime = _connection._pool.Settings._maxResponseDrainTime;
+
```

Comment:
nit: you could move declaration of cts/ctr below this test.

Reply:
I think I'll leave it as is so that the two drainTime comparisons are next to each other.